eth/polar/polaris.go

```diff
@@ -101,6 +107,7 @@ func NewWithNetworkingStack(
 		stack:      stack,
 		host:       host,
 		engine:     host.GetEnginePlugin(),
+		beacon:     &consensus.MockEngine{},
```

Comment:
honestly can rename (to not include "mock") if this is actually being used as beacon